OasisDEX/oasis-borrow
# Incorporated stop loss to aave (just UI)

# [Incorporated stop loss to Aave (just UI)](https://app.shortcut.com/oazo-apps/story/6716/incorporate-stop-loss-to-manage-aave-view)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- added feature toggle `AaveProtection` 
- added protection tab with stoploss (behind feature toggle) to aave manage/view vault page
- adjusted automation context variable names to something more generic
- fixed minor issues with automation form states when vault is empty
  
## How to test 🧪
  <Please explain how to test your changes>

- create aave position (using hardhat), try to enable / disable `AaveProtection` feature toggle
- when feature toggle enable stop loss should be available under protection tab
- for now do not pay attention to stop loss UI content as values was not yet mapped and handled properly as it is in the scope of other task
